nodejs/week1/src/backend/routes/cheap-meals.js

```diff
@@ -0,0 +1,19 @@
+const express = require("express");
+const app = express();
+
+const mealsJson = require("../data/meals.json"); //here you require the data from a JSON file
+const reviewsJson = require("../data/reviews.json"); //here you require the data from a JSON file
+
+// Here you map the meals and matches the meal id with the reviews mealId
+const mealsWithReviews = mealsJson.map(item => {
```

Comment:
Not a big mistake, but you could name them in a bit more appropriate way. For example `meal` instead of `item` and `review` instead of `element`. The same naming can be applied to all other files as well.